### PR TITLE
Add DASH resources to CRM HLD.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**

### CODE REVIEW

File: doc/crm/Critical-Resource-Monitoring-High-Level-Design.md [hunk 1]
```diff
@@ -164,37 +223,78 @@ acl_group_high_threshold                   = 1*4DIGIT                          ;
...
+CRM_STATS_DASH_IPV6_PA_VALIDATION_AVAILABLE      = 1*20DIGIT       ; number of available entries for 'ipv6 pa validation' resource
+CRM_STATS_DASH_IPV6_PA_VALIDATION_USED           = 1*20DIGIT       ; number of used entries for 'ipv6 pa validation' resource
+CRM_STATS_DASH_IPV4_OUTBOUND_CA_TO_PA_AVAILABLE  = 1*20DIGIT       ; number of available entries for 'ipv4 ca to pa' resource
+CRM_STATS_DASH_IPV4_OUTBOUND_CA_TO_PA_USED       = 1*20DIGIT       ; number of used entries for 'ipv4 ca to pa' resource
+CRM_STATS_DASH_IPV6_OUTBOUND_CA_TO_PA_AVAILABLE  = 1*20DIGIT       ; number of available entries for 'ipv6 ca to pa' resource
+CRM_STATS_DASH_IPV6_OUTBOUND_CA_TO_PA_USED       = 1*20DIGIT       ; number of used entries for 'ipv6 ca to pa' resource
+CRM_STATS_DASH_IPV4_DASH_ACL_GROUP_AVAILABLE     = 1*20DIGIT       ; number of available entries for 'ipv4 acl group' resource
+CRM_STATS_DASH_IPV6_DASH_ACL_GROUP_USED          = 1*20DIGIT       ; number of used entries for 'ipv4 acl group' resource
+CRM_STATS_DASH_DASH_IPV4_ACL_GROUP_AVAILABLE     = 1*20DIGIT       ; number of available entries for 'ipv6 acl group' resource
```
[the remark this answers is not in the document]
Comment (oleksandrivantsiv) [reply doc/crm/Critical-Resource-Monitoring-High-Level-Design.md:296]: The description is correct. 64 bites (20 digits) is enough to store the counter for any resource.

[the remark this answers is not in the document]
Comment (zhuhengbo) [reply doc/crm/Critical-Resource-Monitoring-High-Level-Design.md:296]: i mean the description after  1*20DIGIT， Does “ipv4” need to be changed to “ipv6”？

[the remark this answers is not in the document]
Comment (oleksandrivantsiv) [reply doc/crm/Critical-Resource-Monitoring-High-Level-Design.md:296]: Good catch, thank you!

**Review by volodymyrsamotiy — APPROVED** [review]

File: doc/crm/Critical-Resource-Monitoring-High-Level-Design.md [hunk 2]
```diff
@@ -88,6 +101,22 @@ At a high level the following should be supported:
 	- ACL Entries
 	- ACL Counters/Statistics
 10. **FDB entries:** query currently used and available entries
+11. **DASH entries:** query currently used and available entries
+	- VNET
```
Comment (prsunny) [inline doc/crm/Critical-Resource-Monitoring-High-Level-Design.md:105]: Does VNET uses a hardware resource?

Comment (oleksandrivantsiv) [reply doc/crm/Critical-Resource-Monitoring-High-Level-Design.md:105]: Yes, VNET offloads to HW

**Review by prsunny — APPROVED** [review]

**Pull request merged**